examples/v7-next/routes.js

```diff
@@ -0,0 +1,42 @@
+// This file was automatically added by edgio deploy.
+// You should commit this file to source control.
+import { Router } from '@edgio/core/router'
+import { nextRoutes } from '@edgio/next'
+
+// Remove this line to suppress Next's default behavior of removing trailing slashes via a redirect.
+// If trailingSlash: true is set in next.config.js, removing this line will remove the redirect that adds the trailing slash.
+nextRoutes.setEnforceTrailingSlash(true)
+
+export default new Router()
+  // NextRoutes automatically adds routes for all Next.js pages and their assets
+  .use(nextRoutes)
+  .match('/l0-api/:path*', ({ cache, proxy, removeUpstreamResponseHeader, removeRequestHeader }) => {
+    removeRequestHeader('+x-edg-serverless-hint')
+    removeUpstreamResponseHeader('cache-control')
+    cache({
+      edge: {
+        maxAgeSeconds: 60 * 60 * 24,
+        staleWhileRevalidateSeconds: 60 * 60 * 24 * 365,
+      },
+      browser: {
+        maxAgeSeconds: 0,
+        serviceWorkerSeconds: 60 * 60 * 24,
+      },
+    })
+    proxy('api', { path: '/:path*' })
+  })
+  .match('/l0-opt', ({ cache, proxy, removeUpstreamResponseHeader, removeRequestHeader }) => {
+    removeRequestHeader('+x-edg-serverless-hint')
+    removeUpstreamResponseHeader('cache-control')
+    cache({
+      edge: {
+        maxAgeSeconds: 60 * 60 * 24,
+        staleWhileRevalidateSeconds: 60 * 60 * 24 * 365,
+      },
+      browser: {
+        maxAgeSeconds: 0,
+        serviceWorkerSeconds: 60 * 60 * 24,
+      },
+    })
+    proxy('image', { path: '/' })
+  })
```

Comment:
Please update this to use the v7 syntax for the handler instead of the `RouteHelper` usage. The only time we encourage the use of `RouteHelper` is for cases where it's necessary such as `compute` or transform functions. These examples are simple enough to use the JSON format: https://docs.edg.io/guides/v7/performance/cdn_as_code/common_routing_patterns#adding-caching

Reply:
Sure thing!